numpy/doc/subclassing.py

```diff
@@ -555,6 +555,21 @@ def __array_wrap__(self, arr, context=None):
 how this can work, have a look at the ``memmap`` class in
 ``numpy.core``.
 
+Subclassing and Downstream Compatibility
```

Comment:
This section should focus on concrete examples of how NumPy accommodates subclasses and duck-type arrays in its API. For example, the way that numpy functions like `np.sum` attempt to call object methods, and what signature your `sum` method needs to have for this to work. Without context, it's not clear what it means to be "compatible with numpy's method".